ios/ti.animation.xcodeproj/project.pbxproj

```diff
@@ -385,6 +385,7 @@
 				"PROVISIONING_PROFILE[sdk=iphoneos*]" = "";
 				RUN_CLANG_STATIC_ANALYZER = NO;
 				SDKROOT = iphoneos;
+				TARGETED_DEVICE_FAMILY = "1,2,6";
```

Comment:
Does this have to be manually set? Or: Is it actively used in the Ti CLI?

Reply:
without that the catalyst part of the XCFrameworks are not build... (at all for me)

Reply:
Alright!